comixed-rest-api/src/main/java/org/comixed/net/GetAllComicsUnderRequest.java

```diff
@@ -20,21 +20,41 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+/**
+ * <code>GetAllComicsUnderRequest</code> represents the request body during comic file imports.
+ *
+ * @author Darryl L. Pierce
+ */
 public class GetAllComicsUnderRequest {
   @JsonProperty("directory")
   private String directory;
 
+  @JsonProperty("maximum")
+  private Integer maximum;
+
   public GetAllComicsUnderRequest() {}
 
-  public GetAllComicsUnderRequest(final String directory) {
+  public GetAllComicsUnderRequest(final String directory, final Integer maximum) {
     this.directory = directory;
+    this.maximum = maximum;
   }
 
+  /**
+   * The root directory to be searched.
+   *
+   * @return the directory
+   */
   public String getDirectory() {
     return directory;
   }
 
-  public void setDirectory(final String directory) {
-    this.directory = directory;
+  /**
+   * The maximum number of comics to find. If the value is a non-positive valume then all comics
+   * found are returned.
```

Comment:
0 is a positive value in this context ? In French mathematical zero is both positive and negative : in mathematical French "x est positif" (literally "x is positive") allows the case x=0, while "x est positif strictement" (literally "x is strictly positive") does not. There are many other examples in mathematics where "positive" is used to include  0. 
To be clearer, if 0 is not a positive value in this context, it would be better to say if the value is < 1 all comics found will be returned.

Reply:
No. If the value is greater than zero then it's setting a limit on the number of results to return. If the value is zero then there is **no** limit.  The Javadoc may be less than clear in this case.

Reply:
Changed the Javadoc.